src/bipartite.jl

```diff
@@ -72,7 +72,11 @@ end
 
 
 
-LightGraphs.is_directed(b::BipartiteView) = false
+LightGraphs.is_directed(b::BipartiteView{T}) where T = false
+
+LightGraphs.is_directed(::Type{BipartiteView{T}}) where T = false
+
+Base.eltype(::BipartiteView{T}) where T = T
```

Comment:
what is the interpretation of `eltype` in LightGraphs.jl? As I look at:
```
julia> eltype(erdos_renyi(10,0.1))
Int64

julia> eltype(erdos_renyi(UInt8(10),0.1))
UInt8
```
it seems it is the type of the internal storage for vertex numbering. But I might be wrong.

Reply:
It is exactly the type of elements in the adjacency lists of a graph.
In particular it is not edge weight in other libraries such as SimpleWeightedGraphs.jl. 
Hence, in our case the value returned by eltype should be `Int`